conformance/utils/kubernetes/helpers.go

```diff
@@ -210,6 +210,34 @@ func WaitForGatewayAddress(t *testing.T, client client.Client, timeoutConfig con
 	return net.JoinHostPort(ipAddr, port), waitErr
 }
 
+// GatewayMustHaveZeroRoutes validates that the gateway has zero routes attached.  The status may indicate a single listener with not routes or no listeners.
```

Comment:
A couple tiny nits:

- "with not" -> "with zero attached"
- we usually try to wrap comments at 80 chars

Reply:
done